tools/pythonpkg/tests/fast/api/test_read_csv.py

```diff
@@ -540,3 +540,12 @@ def test_read_csv_list_invalid_path(self):
         with pytest.raises(duckdb.IOException, match='No files found that match the pattern "not_valid_path"'):
             rel = con.read_csv(files)
             res = rel.fetchall()
+
+    def test_read_csv_auto_type_candidates(self):
+        con = duckdb.connect()
```

Comment:
Can you add more tests to verify that the types are being passed correctly?
Invalid parameters, checking specific types, and so on.

Reply:
I just though that it's already checked in SQL tests. Double checking stuff is usually a bad idea. But I can add it.

Reply:
It's a different type of check right, what I want to be tested here is that the parameters are being properly converted from the python to the C++ layer, and that all added error messages added are being properly hit.